cuda_core/docs/source/api.rst

```diff
@@ -21,6 +21,8 @@ CUDA runtime
    GraphBuilder
    launch
    Buffer
+   Stream
+   Event
```

Comment:
btw I think this won't work but it's OK to keep them here. I will fix them in #695 (I haven't pushed my local changes).

Reply:
It surprisingly works... 🤔